ui/server/src/main/scala/pl/touk/nussknacker/ui/validation/PrettyValidationErrors.scala

```diff
@@ -24,7 +25,7 @@ object PrettyValidationErrors {
       case InvalidRootNode(_) => node("Invalid root node", "Process can start only from source node")
       case InvalidTailOfBranch(_) => node("Invalid end of process", "Process branch can only end with sink or processor")
 
-      case MissingParameters(params, ProcessCompilationError.ProcessNodeId) =>
+      case MissingParameters(params, NodeTypingInfo.ExceptionHandlerNodeId) =>
```

Comment:
Hmm... I wonder if these are only ExceptionHandler params...

Reply:
I also was wonder but than I found usage of `ProcessNodeId` and it was only in one place: `ProcessCompiler.compile(ref: ExceptionHandlerRef)`. I know that will be better to have `MissingParametersInExceptionHandler`, but during this feature there was a lot of refactors... Maybe we will do some in other one? ;-)